### DISCUSSION TITLE
Polymorphic relation GUI

### DISCUSSION THREAD

**Discussion by ballesta** [TiagoSilvaPereira/vemto-issues#179] [Q&A]
Hi,

In my "polymorph" project i have 3 entities:

- fields 
- integernumber
- date

I try to implement a Polymorphic relation between : 

- integernumber and field
- date and field
using a Morph One Polymorphic relationship.

The attributes fieldable_id and fieldable_type are automatically added to the field entity.

hat i would like upon adding a new field:

- Have a drop down box or radio button to select the type of the field (integer  or date)
- Have a subform allowing to input the attributes of the chosen field type (from, to for date) 
  
Can you see the  schema in my project?

Best regards 
Bernard

**Comment by TiagoSilvaPereira** [comment]
Hi @ballesta GUI for polymorphic relationships is in my plans for the future... thank you for your suggestion
